frontend-react/.env.production

```diff
@@ -1,8 +1,17 @@
-; production overrides .env values
+; this is set by env var being set `CLIENT_ENV: production` in the github action
+REACT_APP_TITLE=CDC Prime ReportStream
+REACT_APP_DESCRIPTION=ReportStream Dashboard Application
+REACT_APP_ICON=https://reportstream.cdc.gov/assets/cdc-logo.svg
+
 REACT_APP_BASE_URL=https://reportstream.cdc.gov
```

Comment:
I think this now needs to be `https://prime.cdc.gov` since that is what is used in AuthResource for production

Reply:
Hmm, we may wanna alter that to use `REACT_APP_BACKEND_URL` and just set that value to `localhost:3000` in `.env.development`. That would still adhere to the `proxy` setup we have in local and be a more clear naming convention since really we _aren't_ using the React app base url for those calls.

Reply:
This variable is used by `getBaseUrl()`

Here's the original function
```javascript

    static getBaseUrl = () => {
        if (window.location.origin.includes("localhost"))
            return "http://localhost:3000";
        else if (window.location.origin.includes("staging"))
            return "https://staging.prime.cdc.gov";
        else if (window.location.origin.includes("test"))
            return "https://test.reportstream.cdc.gov";
        else return "https://prime.cdc.gov";
    };
```
So, you are correct, it looks like it should be `https://prime.cdc.gov` for prod

Excellent catch!